dist/svg.select.js

```diff
@@ -16,13 +16,41 @@ function SelectHandler(el) {
     this.pointSelection = {isSelected: false};
     this.rectSelection = {isSelected: false};
 
+    // helper list with position settings of each type of point
+    this.pointsList = {
+      lt: [ 0, 0 ],
+      rt: [ 'width', 0 ],
+      rb: [ 'width', 'height' ],
+      lb: [ 0, 'height' ],
+      t: [ 'width / 2', 0 ],
+      r: [ 'width', 'height / 2' ],
+      b: [ 'width / 2', 'height' ],
+      l: [ 0, 'height / 2' ]
+    };
+
+    // helper function to evaluate point coordinates based on settings above and an object (bbox in our case)
+    this.pointCoord = function (setting, object) {
+      return typeof setting !== 'string' ? setting : eval('object.' + setting)
```

Comment:
I will ignore that use of `eval` here since it seems to really reduce the needed code :D

Reply:
Yeah, I agree that's NOT a good practice, but I had to come up with a smart idea for defining a config object for the points ;)

Anyway, it's evaluating safe code (not user input)

Reply:
I agree. I think its fine

Reply:
As there are not that many options, I think I have another solution to replace the `eval` thing.
As soon as I find half an hour to implement & test, I will let you know, maybe we'll change it.

PS: Maybe there's a good idea to update the version as well, or why on `npm` is still loading the old version ?

Reply:
I didnt published a new version yet.
When its not out tomorrow remind me again :)